portal/server/helpers/dealFormsCompleted.js

```diff
@@ -1,21 +1,24 @@
 const CONSTANTS = require('../constants');
 
-
 /**
  * Checks if a facility is complete by filtering its items based on their status and certain conditions,
  * and then comparing the length of the filtered arrays with the total number of items.
- * @param {Object} facility - The facility object containing an array of items.
+ * @param {Object} facilities - The facility object containing an array of items.
  * @returns {boolean} - True if the facility is complete, false otherwise.
  */
-const isFacilityComplete = (facility) => {
-  const items = facility?.items || [];
+const isFacilityComplete = (facilities = []) => {
```

Comment:
It is strange mix of plural and singular names. Before it was checking some `items` and now it is called `facilities`?

if function is checking facilities, should it be called `isDealComplete`
if function is checking facility subitems, maybe input variable should be `items`/`subitems`

Reply:
I was commenting on old code, marking as resolved.

Reply:
I have changed it to `facilities` to accurate define the array of facilities as an argument, beforehand we were sending `deal.bondTransactions` and `deal.loanTransactions` separately with `.items` as an Array.

I have now merged these arrays together, as all these facilities belong to the same deal.

No this function only checks for the facilities status, `isDealComplete` would not accurately represent this function as it involves checking EC, submission details and etc which is covered by `isEveryDealFormComplete`.